docs/components/MultiSelect/MultiSelect.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Canvas, Meta } from "@storybook/addon-docs";
+import { useState } from "react";
+import { Divider } from "@jobber/components/Divider";
+import { Content } from "@jobber/components/Content";
```

Comment:
Are these imports still required?

Reply:
I'm the worst at getting rid of these imports 😅  thank youuu I'll get rid of them!